solar_system.rb

```diff
@@ -0,0 +1,208 @@
+
+mercury =
+  {
+    name: "Mercury",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+venus =
+  {
+    name: "Venus",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+  earth =
+  {
+    name: "Earth",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+  mars =
+  {
+    name: "Mars",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+  jupiter =
+  {
+    name: "Jupiter",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+  saturn =
+  {
+    name: "Saturn",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+  uranus =
+  {
+    name: "Uranus",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+
+  neptune =
+  {
+    name: "Neptune",
+    distance_from_sun: 100,
+    color: "blarb",
+    moons: false,
+    size: "large",
+    sustains_life: false
+  }
+# instead of strings for planets, modify initialize method to take a list of hashes where each planet is sent as a hash with at least 5 attributes
+planet_names = [mercury, venus, earth, mars, jupiter, saturn, uranus, neptune]
+#new_planet = "Hoth"
+hoth =
+{
+  name: "Hoth",
+  distance_from_sun: 100,
+  color: "blarb",
+  moons: false,
+  size: "large",
+  sustains_life: false
+}
+new_planet = hoth
+
+class SolarSystem
+
+  def initialize(planet_names)
+    @planets = planet_names
+    #puts "Initializing beep boop"
+  end
+
+  def add_planet(new_planet)
+    @new_planet = new_planet
+    @planets.push(new_planet)
+  end
+
+  def return_planets
+    planet_strings = "" #this is the string that will accumulate my return strings from the loop and then split them apart by new line
+    idx = 1
+    @planets.each do |planet|
+        planet_strings += "#{idx}. #{planet[:name]} \n"
+        idx += 1
+    end
+    return planet_strings
+
+  end
+
+end
+
+class Planet
+  attr_reader :name, :distance_from_sun, :color, :moons, :size, :sustains_life
+
+  def initialize(name, distance_from_sun, color, moons, size, sustains_life)
+    @name = name
+    @distance_from_sun = distance_from_sun
+    @color = color
+    @moons = moons
+    @size = size
+    @sustains_life = sustains_life
+  end
+
+  planet_summary = ""
+  def summarize_planet
+    planet_summary = "Name: #{@name}\n" +
+                     "Distance from Sun: #{@distance_from_sun}\n" +
+                     "Color: #{@color}\n" +
+                     "Moons: #{@moons}\n" +
+                     "Size: #{@size}\n" +
+                     "Sustains Life: #{@sustains_life}"
```

Comment:
Style note: you should have an explicit return here. Adding in the line `return planet_summary` would make the code a little more readable.